### PR TITLE
Book details page

### PR DESCRIPTION
**Pull request by mwharrison**

### CODE REVIEW

**Comment by mwharrison** [comment]
[OSW2.refresh.14 Update book details page](https://trello.com/c/9DDGbXnC/539-osw2refresh14-update-book-details-page)

File: books/migrations/0040_auto_20180402_1158.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='comp_copy_blurb',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='cover_color',
+            field=models.CharField(choices=[('Blue', 'Blue'), ('Deep Green', 'Deep Green'), ('Gold', 'Gold'), ('Gray', 'Gray'), ('Green', 'Green'), ('Light Blue', 'Light Blue'), ('Light Gray', 'Light Gray'), ('Medium Blue', 'Medium Blue'), ('Orange', 'Orange'), ('Red', 'Red'), ('Yellow', 'Yellow')], default='Blue', max_length=255),
```
Comment (RoyEJohnson) [inline books/migrations/0040_auto_20180402_1158.py]: Is this a list of label-value pairs? Can the value be lower-case and hyphenated? That matches the values in the pattern library (e.g. `medium-blue`).

File: books/models.py [hunk 2]
```diff
@@ -457,8 +538,17 @@ def get_student_handbook_url(self):
                   'high_resolution_pdf_url',
                   'low_resolution_pdf_url',
                   'student_handbook_url',
+                  'free_stuff_heading',
+                  'free_stuff_blurb',
                   'community_resource_url',
                   'community_resource_cta',
+                  'community_resources_blurb',
+                  #'community_resources_blurb',
```
Comment (RoyEJohnson) [inline books/models.py]: Did you want this comment left in?

**Comment by mwharrison** [comment]
@RoyEJohnson - I made that commented out field a document upload for now (I think that's what they want), I'll change if necessary. I also updated the API output to be lowercase to match pattern library.

**Pull request merged**